packages/common/sagas/entities/team.js

```diff
@@ -128,11 +138,38 @@ export function* createTeamSaga({ payload: { title, icon } }) {
     const { data } = yield call(postCreateTeam, { title, icon });
 
     const user = yield select(userDataSelector);
+    const defaultList = yield select(defaultListSelector);
+    const masterPassword = yield call(passwordGenerator);
+    const systemTeamEmail = yield call(generateSystemItemEmail, data.id);
+
+    const {
+      publicKey,
+      privateKey,
+    } = yield call(generateKeys, masterPassword, [systemTeamEmail]);
 
     yield put(createTeamSuccess({ ...data, __type: ENTITY_TYPE.TEAM }));
     yield put(addTeamToMember(data.id, user.id));
```

Comment:
please, review the lines from 151 to 153, maybe it's redundant.

Reply:
Renamed actions